docs/getting-started/deploy-function.md

```diff
@@ -0,0 +1,58 @@
+# Deploying a Knative Function
+
+Deploy your function using the `func deploy` command. When deploying, your
+function source code is built into an OCI container image, pushed to a
+container registry and then deployed to your cluster as a Knative Service.
+
+Deploying from a local build requires that you have a Docker daemon running
+on your local computer. Additionally, you must have a Docker registry that you
+can push to. Specify the registry using the `--registry` flag. Typically the
+registry is specified as `<registry>/<username>`, for example
+`docker.io/exampleuser`.
+
+This command uses the function project name and the image registry name to
+construct a fully qualified image name for your function.
+
+Deploy the function by running the command specifying the project directory:
+
+```{ .console }
+func deploy --registry <registry> -p hello
```

Comment:
I would remove the extra flag `-p hello` and have the user work inside the `hello` directory.